docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.4"
+
+services:
+  localstack-s3:
+    image: localstack/localstack:latest
+    container_name: localstack-s3
+    environment:
+      - SERVICES=s3:5002
+      - PORT_WEB_UI=8081
+      - DEFAULT_REGION=eu-west-2
+      - DATA_DIR=/tmp/localstack/data
+    ports:
+      - "5002:5002"
+      - "8081:8081"
+
+  db:
+    image: postgres
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: example
+      POSTGRES_DB: airflow-db
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 30s
+      timeout: 30s
+      retries: 3
+
+  app:
+    build:
+      dockerfile: Dockerfile
+      context: .
+      args:
+        - env=dev
+    volumes:
+      - ./airflow.cfg:/src/airflow/airflow.cfg
```

Comment:
a sizable file to provide from outside; any options for making it smaller or shipping a default `.cfg` file in the image? Not sure how many of the values in there are local (paths to folders) and stable vs. remote (host names that will change across environments)

Reply:
This is automatically generated. I'm only changing a few values. An alternative is to use environment variables to set configuration values. I could try removing all of the values I'm not using but I don't what the effects of this would be.

Reply:
Main value that stands out is the database connection parameters:
https://github.com/libero/elife-style-content-adapter-prototype/pull/1/files#diff-ea36df1840fe05258ac3690974a20733R53
which means the file cannot be shipped inside the image. The existing standard is to put these configuration files into a [`.docker/` subfolder](https://github.com/libero/browser/tree/master/.docker) so that it's clear what is needed to [run the image elsewhere in a non-`dev` environment](https://github.com/libero/sample-configuration).

Reply:
Bear in mind the title of the PR: `Dockerize dev environment`. I'll update this to use environment variables which will only be present in the dev image.

Reply:
updated.

Reply:
> I'll update this to use environment variables 

That's good, now that I am aware of the possibility.